integritybackend/actions.py

```diff
@@ -376,6 +376,7 @@ def archive(self, zip_path: str, org_config: dict, collection_id: str):
                         "application/octet-stream",
                         meta_content["dateCreated"],
                         asset_extras,
+                        ["numbers", "avalanche", "near"],
```

Comment:
This is hardcoded? I thought this is selectable per collection.

Reply:
With the old API, the single register call would register the asset on both Numbers and Avalanche chains. So I retained that behaviour, just at the action level instead of in the register code. I can make it configurable through the org config if that's what we want.